calico/felix/actor.py

```diff
@@ -508,6 +509,8 @@ def queue_fn(self, *args, **kwargs):
             if (not on_same_greenlet and
                     not async and
                     _log.isEnabledFor(logging.DEBUG)):
+                    # MD4: I don't understand what the point of this
```

Comment:
Ah, this has been refactored (maybe by Peter?).  I think the calling path lookup used to be done after this guard but I see we're not using that for something else now so it's no longer needed.